### PR TITLE
Build connector functionality for git pipelines

### LINKED ISSUE #117: Implement "connector" output functionality for "standard" pipelines as well
Standard pipelines only have one redis output available. For this reason we introduced "connector pipelines" to build more sophisticated setup. We came up with a way of optinally having your own inputs and outputs so connectors lost importance. We can implement the `if` logic we had with connectors for all other pipelines as well, thus giving maximum flexibility and provide a lot cleaner code.

### PR DESCRIPTION
**Pull request by widhalmt**
fixes #117

### CODE REVIEW

**Comment by afeefghannam89** [comment]
The loop want a list but the - is not available  before the syslog word that is why Ansible can not read the **logstash_pipelines**  as a list he see it as a dictionary which does not suit the loop way https://github.com/NETWAYS/ansible-role-logstash/blob/e15a9f17817379b117bd137cfce80776ccda0f24/molecule/run_logstash/converge.yml#L18

File: templates/simple-output.conf.j2 [hunk 1]
```diff
@@ -1,7 +1,20 @@
 output {
+{% for output in item.output %}
+
+# {{ output.name }} output
+{% if output.condition is defined or item.exclusive | bool %}
+{% if not loop.first and item.exclusive | bool %}else {% endif %}{% if (not loop.last and item.exclusive | bool) or not item.exclusive %}if {% endif %}{% if output.condition is defined %}{{ output.condition }} {% endif %}{
```
Comment (afeefghannam89) [inline templates/simple-output.conf.j2:6]: Maybe there is here extra if?

**Review by afeefghannam89 — COMMENTED** [review]
ohhh too much good changes :)

File: templates/simple-output.conf.j2 (hunk 1 above)
Comment (widhalmt) [reply templates/simple-output.conf.j2:6]: No, it seems to work that way. What's especially tricky about this part of the code is that there are mixed Ansible `if` and Logstash `if`.

**Pull request merged**